R/cfr.R

```diff
@@ -117,11 +117,16 @@ cfr_ui <- function(id, full_screen = TRUE) {
             ),
             parameter_tabs
           ),
-          actionButton("go", "Get estimate")
+          actionButton(
+            inputId = NS(id, "go"),
```

Comment:
Thanks @PaulC91 - I see that it was missing this wrapping of the `inputId` in `NS()`.

Reply:
Yeah that was it! I've lost many hours of my life debugging to then realise I forgot to wrap an id in a namespace function 😅

Reply:
Thanks for spotting!